src/Endpoints/Payments.php

```diff
@@ -257,7 +265,7 @@ private function doRefund(Refund $refundPayload) {
 
         $res = $req->post();
         if ($res->isError()) {
-            throw new RequestError($res->errorMessage, $req, $res);
+            throw new RequestException($res->errorMessage, $req, $res);
```

Comment:
don't change the type of exception for now, because it implies we change the catch in all cms :) cc @Francois-Gomis